lib/rails_admin/config/actions/base.rb

```diff
@@ -78,6 +78,11 @@ class Base
           true
         end
 
+        # Target window [_self, _blank]
+        register_instance_option :target_window do
```

Comment:
To me, `link_target` feels like to be a better name. What do you think?

Reply:
Yes, makes sense! why not